SPECS/nodejs/nodejs.spec

```diff
@@ -1,56 +1,91 @@
 # Retrieved from 'deps/npm/package.json' inside the sources tarball.
-%define npm_version 6.14.13
+%define npm_version 8.3.1
 
 Summary:        A JavaScript runtime built on Chrome's V8 JavaScript engine.
 Name:           nodejs
 # WARNINGS: MUST check and update the 'npm_version' macro for every version update of this package.
 #           The version of NPM can be found inside the sources under 'deps/npm/package.json'.
-Version:        14.17.2
-Release:        2%{?dist}
+Version:        16.14.0
+Release:        1%{?dist}
 License:        BSD and MIT and Public Domain and naist-2003
 Group:          Applications/System
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://github.com/nodejs/node
+# !!!! Nodejs code has a vendored version of OpenSSL code that must be removed from source tarball 
+# !!!! because it contains patented algorithms.
+# !!!  => use clean-source-tarball.sh script to create a clean and reproducible source tarball.
```

Comment:
I'd also add the script as `Source1` like we do in 1.0. This way, it's also going to be available with the SRPM.

Reply:
why would we need it in SRPM ? the script is in github and used to create the tarball that ends up in SRPM

Reply:
Hmmm, I guess you're right, that it's not part of sources required for building. Other distros seem to pack similar scripts with their SRPMs (look for `generate-tarball.sh` in the specs), but I'm OK with not following that.